models/repo_tag.go

```diff
@@ -14,6 +14,7 @@ func GetTagsByPath(path string) ([]*git.Tag, error) {
 	if err != nil {
 		return nil, err
 	}
+	defer gitRepo.Close()
 
 	return gitRepo.GetTagInfos()
```

Comment:
This could be a problem since Tag contains a reference to Repository.
```
type Tag struct {
	Name    string
	ID      SHA1
	repo    *Repository
	Object  SHA1 // The id of this commit object
	Type    string
	Tagger  *Signature
	Message string
}
```

Reply:
Probably is.

I'll take another look.

Reply:
Oh dear GetTagInfos is a terrible implementation. It needs rewriting...